src/CMakeLists.txt

```diff
@@ -637,9 +637,23 @@ install(TARGETS ceph-mon DESTINATION bin)
 if(NOT ALLOCATOR STREQUAL "jemalloc")
   set(disable_jemalloc "DISABLE_JEMALLOC=1")
 endif()
+
+set(ROCKSDB_EXTRA_CXXFLAG -fPIC -Wno-unused-variable)
+if(NOT CMAKE_BUILD_TYPE MATCHES "Debug")
+  set(ROCKSDB_EXTRA_CXXFLAG ${ROCKSDB_EXTRA_CXXFLAG} -DNDEBUG)
+endif(NOT CMAKE_BUILD_TYPE MATCHES "Debug")
+
+# get number of CPUs to build rockdb with
+if(LINUX) 
```

Comment:
i am not sure about this change, as 

1. we plan to use cmake to build rocksdb once https://github.com/facebook/rocksdb/pull/1135 is merged.
1. rocksdb is not changed very frequently in comparison to other components in ceph.
1. the heuristics of "nprocs/2" might not work well if we are already `make -j $(($(nproc)/2))`